BeeSwift/GoalViewController.swift

```diff
@@ -63,9 +72,20 @@ class GoalViewController: UIViewController,  UIScrollViewDelegate, DatapointTabl
         self.dateStepper.minimumValue = -365
         self.dateStepper.maximumValue = 365
 
+        self.view.addSubview(self.timeElapsedView)
+        self.timeElapsedView.snp.makeConstraints { (make) -> Void in
+            make.top.equalTo(self.view.safeAreaLayoutGuide.snp.topMargin)
+            make.left.equalTo(0)
+            make.right.equalTo(0)
+        }
+        
+        self.updateLastUpdatedLabel()
+        Timer.scheduledTimer(timeInterval: 60, target: self, selector: #selector(GoalViewController.updateLastUpdatedLabel), userInfo: nil, repeats: true)
```

Comment:
[Consider] Wouldn't this timer make more sense being part of the `FreshnessIndicatorView` ?

Reply:
The reason I placed it here is because this is also a Controller whereas the Freshness one is just a view.